### DISCUSSION TITLE
How can send  message with  mention  a role?

### DISCUSSION THREAD

**Discussion by arifurrahmansw** [spatie/laravel-discord-alerts#28] [Q&A] [answered]
![image](https://github.com/spatie/laravel-discord-alerts/assets/17975092/a0cbaaa8-493a-4e1e-8d87-94f76f7f309b)

**Comment by Baspa** [comment]
I'm curious about this too, is this possible?

**Comment by ghost** [comment, marked as answer]
Figured it out. Find the userid or roleid (enable developer mode in Discord) and use `<@userid>` or `<&roleid>`

**Reply by arifurrahmansw** [reply]
Thanks
